projectroles/tests/test_templatetags.py

```diff
@@ -725,3 +725,184 @@ def test_get_sidebar_links_timeline_view(self):
                     self.assertEqual(app['active'], True)
                 else:
                     self.assertEqual(app['active'], False)
+
+    def test_get_user_links_home(self):
+        """Test get_user_links() on the home view"""
+        url = reverse('home')
+        with self.login(self.user):
```

Comment:
With `RequestFactory`, using `self.login()` does nothing, as we create the request manually. Hence this can be removed from all tests.